tests/fixtures/db.js

```diff
@@ -21,33 +24,54 @@ const sampleUsers = [
   }
 ]
 
+// cache tokens from sample user logins here
+const tokens = []
+
 const syncDatabase = async () => {
   // for each model: drop table if exists, then create new table
   await db.sync({ force: true })
 }
 
 const populateTables = async () => {
-  // delete all rows in table
+  // delete all rows in tables
   await User.destroy({ where: {} })
+  await Post.destroy({ where: {}})
+
+  // clear cache of tokens
+  tokens.length = 0
 
   await Promise.all(
     // loop thru sampleUsers
     sampleUsers.map(async user => {
       // cache the hashed passwords
       const hashedPassword = await bcrypt.hash(user.password, 8)
 
-      // insert sample users to DB with the hashed passwords
+      // insert sample user to DB with the hashed passwords
       await User.create({
         name: user.name,
         email: user.email,
         password: hashedPassword
       })
+
+      // login sample user
+      const loginResponse = await request(app)
+        .post('/users/login')
+        .send({
+          email: user.email,
+          password: user.password
+        })
+
+      const token = loginResponse.body.token
+
+      // add login token to array
+      tokens.push(token)
```

Comment:
I am not sure if I should be using the actual api endpoint to do login in the db setup here. The other alternative is to just use the jsonwebtoken module to just create new token (which is effectively what the login route does, and it also matches the login credentials to a user in the db).

It seemed like a good idea to do it this way because that would most closely simulate how an actual user login would work. I don't see it being a problem, but if it is bad practice or interferes we can switch to just creating the token.